clicommand/cancel_signal.go

```diff
@@ -29,6 +25,6 @@ var (
 		Usage: "The number of seconds given to a subprocess to handle being sent ′cancel-signal′. " +
 			"After this period has elapsed, SIGKILL will be sent.",
```

Comment:
The Usage string for this flag should be updated to describe the new interpretation of -1 (or negative values generally).